app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
+    masks_to_buy = 0
+    is_vaccine = True
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            is_vaccine = False
```

Comment:
You can return `'All friends should be vaccinated'` message here and not use `is_vaccine` variable
But, this solution is ok anyway)

Reply:
okay, thx!